frontend/tailwind.config.js

```diff
@@ -23,7 +24,34 @@ export default {
                 200: '#1C2227',
             },
         },
-        extend: {},
+        fontSize: {
```

Comment:
Could you please double-check the `rem` values? It seems like something might be slightly off.

Reply:
Thanks for noticing! Really something went slightly different. Now fixed